BugBounty/src/test/java/com/revature/service/ApproveDenyBugServiceTest.java

```diff
@@ -0,0 +1,52 @@
+package com.revature.service;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.MockitoAnnotations;
+import org.testng.annotations.BeforeSuite;
+import org.testng.annotations.Test;
+
+import com.revature.model.BugReport;
+import com.revature.model.Role;
+import com.revature.model.User;
+import com.revature.repository.BugReportRepository;
+
+public class ApproveDenyBugServiceTest {
+	@Mock
+	BugReportRepository bugReportRepository;
+	
+	@InjectMocks
+	BugReportService bugReportService;
+	
+	private User user = new User(6, "Bob", "password", 40, new Role(1, "user"));
+	private BugReport bugReport = new BugReport(8, user, "Test", "Test", "Test", "Test", "Test", null, "open");
+	private List<BugReport> pendingBugReports =  new ArrayList<>();
+	
+	@BeforeSuite
+	public void setUp() {
+		
+		bugReportService = new BugReportService();
+		//initialize Mockito
+		MockitoAnnotations.initMocks(this);
+	}
+	
+	@Test
+	public void testUpdateBugReportStatus() {
+		//Mockito.doNothing().when(bugReportRepository).updateStatus(8, "open");
+		//Mockito does nothing the first time for free
+		bugReportService.updateBugReportStatus(bugReport);
+	}
+	
+	@Test
```

Comment:
Is there some business logic present in the deleteBugReport method in your service layer that you're intending to test here?

Reply:
I was wanting to test that the service layer calls the repository deleteById. Wasn't sure how to mock that. Is there a btter way, same with the UpdateBugReportStatus?

Reply:
You may want to use Mockito's "verify" for this task.

Reply:
Ok I will resubmit